src/superphot_plus/samplers/licu_sampler.py

```diff
@@ -0,0 +1,212 @@
+    left_bound[1] = -1e-3 * prior_clip_a[0]
+
+    right_bound = transform_to_licu(*prior_clip_b)
+    # change baseline to be not exactly zero
+    right_bound[1] = 1e-3 * prior_clip_a[0]
+    # relative plateau amplitude must be less than 1
+    right_bound[5] = np.min([right_bound[5], 0.99])
+
+    initial_guess = transform_to_licu(*prior_mean)
+
+    villar_fit = licu.VillarFit(  # pylint: disable=no-member
+        init=initial_guess,
+        bounds=list(zip(left_bound, right_bound)),
+        **licu_kwargs
+    )
+
+    # We have nothing better to do than just use the mean of the prior
+    extra_sigma = prior_mean[6]
+
+    # light-curve package parameters
+    # Reduced Chi^2 definition is different from the one in superphot+
+    # *features, _reduced_chi2 = villar_fit(lc.times, lc.fluxes, np.hypot(lc.flux_errors, extra_sigma))
+    try:
+        *features, _reduced_chi2 = villar_fit(lc.times, lc.fluxes, lc.flux_errors)
+    except ValueError:
+        features = initial_guess
+
+    cube = transform_from_licu(*features)
+    # Use mean of prior for extra_sigma
+    cube[6] = extra_sigma
+
+    return cube
+
+
+def transform_priors_to_physical_values(cube, *, max_flux, reference_idx):
+    """Some priors are for log-params, this transforms them back to linear"""
+    cube = cube.reshape(-1, 7).copy()
+
+    # Scale log-priors to linear values
+    cube[:, [0, 2, 4, 5, 6]] = 10 ** cube[:, [0, 2, 4, 5, 6]]
+
+    # Non-reference band priors must be scaled by the reference band prior
+    non_ref_idx = np.delete(np.arange(cube.shape[0]), reference_idx)
+    cube[non_ref_idx] *= cube[reference_idx]
+
+    # amplitude and extra_sigma must be scaled by the maximum flux
+    cube[:, [0, 6]] *= max_flux
+
+    return cube.reshape(-1)
+
+
+def transform_to_licu(amp, beta, gamma, t_0, tau_rise, tau_fall, extra_sigma):
+    """Transforms superphot+ parameters to light-curve package parameters"""
+    del extra_sigma  # no extra_sigma in light-curve package
+    amplitude = amp
+    baseline = 0.0  # no baseline in superphot+
+    reference_time = t_0
+    rise_time = tau_rise
+    fall_time = tau_fall
+    plateau_rel_amplitude = beta * gamma
+    plateau_duration = gamma
+    return np.array([
+        amplitude,
+        baseline,
+        reference_time,
+        rise_time,
+        fall_time,
+        plateau_rel_amplitude,
+        plateau_duration,
+    ])
+
+
+def transform_from_licu(
+        amplitude,
+        baseline,
+        reference_time,
+        rise_time,
+        fall_time,
+        plateau_rel_amplitude,
+        plateau_duration,
+):
+    """Transforms light-curve package parameters to superphot+ parameters"""
+    del baseline  # no baseline in superphot+
+    amp = amplitude
+    beta = plateau_rel_amplitude / plateau_duration
+    gamma = plateau_duration
+    t_0 = reference_time
+    tau_rise = rise_time
+    tau_fall = fall_time
+    extra_sigma = 1e-3  # no extra_sigma in light-curve package
```

Comment:
maybe make this the consistent with the prior mean instead of 1e-3?

Reply:
It is actually overwritten by the actual value in `fit_single_band`